### ISSUE TITLE
What's the Norm function?

### ISSUE THREAD

**Issue by iamhankai** [zhoudaquan/dvit_repo#1]
What's the Norm function in Eq.(3)? LayerNorm?

**Comment by zhoudaquan** [comment]
> What's the Norm function in Eq.(3)? LayerNorm?

We use batch norm for the experiments in the paper.

**Comment by iamhankai** [comment]
Thanks a lot.

**Comment by iamhankai** [comment]
Another question: for the input `BxHxNxN`, BN is performed along which dim?

**Comment by zhoudaquan** [comment]
> Another question: for the input `BxHxNxN`, which dim is BN performed along ?

Hi, it is applied along the batch dimension.

**Comment by iamhankai** [comment]
`nn.BatchNorm2d(num_features=H)` or `nn.BatchNorm2d(num_features=N)`?

**Comment by ggjy** [comment]
Good work！What about the initialization of the `HxH` matrix？Is `torch.eye` or `torch.randn`?

**Comment by zhoudaquan** [comment]
> Good work！What about the initialization of the `HxH` matrix？Is `torch.eye` or `torch.randn`?

Hi,

Thanks for your interest. This is a good question, I try both the eye init and random init and the results are similar. For the experiments in the paper, the results are based on random init. I am considering add in a set of experiments regarding the initialization into the paper also.

Best regards,
Zhou Daquan

**Comment by iamhankai** [comment]
Thank you very much.
